test/unit/test-timer.cpp

```diff
@@ -84,15 +88,14 @@ TEST(TimerTest, No2)
 
   for (int i = 2; i > 0; --i) {
     std::cout << i << std::endl;
-    std::this_thread::sleep_for(std::chrono::seconds(1));
+    std::this_thread::sleep_for(std::chrono::milliseconds(10));
   }
 
   timer.stop();
 
   RAJA::Timer::ElapsedType elapsed = timer.elapsed();
 
-  EXPECT_TRUE(elapsed > 2.0);
+  EXPECT_GT(elapsed, 0.02);
+  EXPECT_LT(elapsed, 0.05);
```

Comment:
It seemed high enough to be clearly safe, while low enough to be sure the timer isn't returning random values (I'm looking at you BlueGene/Q 👈 ).  I have no issue with making it higher, but do rather want to have an upper bound just as a sanity check.